float32 cart_position
float32 cart_velocity
float32 pole_angle
float32 pole_angular
int16 linear_x
float32 eta
uint32[] particles_pos
